bentoml/utils/usage_stats.py

```diff
@@ -155,10 +155,8 @@ def track_load_finish(bento_service):
     return track("load", properties)
 
 
-def track_cli(command, deploy_platform=None, extra_properties=None):
+def track_cli(command, extra_properties=None):
```

Comment:
this function doesn't do anything now, let's remove `track_cli` and call `track` directly in the command group code?